### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mellis13**
This PR updates the README content and links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+part of the SmartSim documentation.
+
+## Dependencies
+
+SmartRedis utilizes the following libraries.
+
+ - [NumPy](https://github.com/numpy/numpy)
+ - [Hiredis](https://github.com/redis/hiredis) 1.0.0
+ - [Redis-plus-plus](https://github.com/sewenew/redis-plus-plus)  1.2.3
+ - [protobuf](https://github.com/protocolbuffers/protobuf)  v3.11.3
```
Comment (al-rigazzi) [inline README.md:64]: Is there a reason why you put `v` for Protobuf version, but not for all other libraries?

**Review by al-rigazzi — APPROVED** [review]
Looks good to me

**Comment by Spartee** [comment]
Some conflicts but otherwise LGTM.

File: README.md (hunk 1 above)
Comment (mellis13) [reply README.md]: Great catch!  Removed the inconsistent 'v'.

**Pull request merged**